### PR TITLE
GA migration Postcommit Python Sickbay

### PR DESCRIPTION
**Pull request by benWize**
As part of the migration of Precommit and Postcommit Jobs from Jenkins to GA in self-hosted runners, this PR contains:

- Migrated workflow [job-postcommit-python-sickbay.yml](https://github.com/benWize/beam/blob/24ef2f9502babf737304753ecfdefd5538fa58c2/.github/workflows/job-postcommit-python-sickbay.yml)

The migrated workflows were removed from [.test-infra/jenkins/README.md](https://github.com/benWize/beam/blob/5716ca8c31a1e0046ca91472b0aafad220f57567/.test-infra/jenkins/README.md) and added to [CI.md](https://github.com/benWize/beam/blob/24ef2f9502babf737304753ecfdefd5538fa58c2/CI.md)

**DO NOT MERGE** until the effort to use **self-hosted runners** is completed https://github.com/apache/beam/pull/22703

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by benWize** [comment]
Failed correctly executions on self-hosted runners:

Sickbay 3.7, 3.8, 3.9: https://github.com/benWize/beam/actions/runs/3041163675

**Comment by fernando-wizeline** [comment]
LGTM!

File: CI.md [hunk 1]
```diff
@@ -125,6 +125,13 @@ Service Account shall have following permissions ([IAM roles](https://cloud.goog
 | Java Wordcount Direct Runner | Runs Java WordCount example with Direct Runner.                                               | Yes              | Yes                   | Yes           | -                        |
 | Java Wordcount Dataflow      | Runs Java WordCount example with DataFlow Runner.                                             | -                | Yes                   | Yes           | Yes                      |
 
+### PostCommit Workflows
+
+#### Python Sickbay - [job-postcommit-python-sickbay.yml](.github/workflows/job-postcommit-python-sickbay.yml)
```
Comment (damccorm) [inline CI.md:130]: As mentioned in other PRs, please make this part of a single table

Comment (benWize) [reply CI.md]: Done!

**Review by damccorm — APPROVED** [review]
LGTM

**Pull request closed without merging**